lib/applications.js

```diff
@@ -61,9 +62,11 @@ exports.createApplication = async (req, res) => {
 
         // We don't need to recalculate the votes amount, as the pax type is not set here.
 
+        newApplication.notification_email = await core.fetchUser(req.body, req.headers['x-auth-token']);
```

Comment:
this request is also not needed, as it can be taken from req.user, right?

Reply:
True, will fix